okhttp/src/main/java/okhttp3/internal/http/CallServerInterceptor.java

```diff
@@ -131,4 +145,28 @@ public CallServerInterceptor(boolean forWebSocket) {
 
     return response;
   }
+
+  private long headerLength(Headers headers) {
+    long length = 0;
+
+    for (int i = 0, size = headers.size(); i < size; i++) {
```

Comment:
Should we add 4 bytes overhead per header for `: ` and `\r\n` ? I’m tempted to

Reply:
Your call. I wouldn't bother as it seems wrong anyway for HTTP/2 with a binary hpack encoding.